### ISSUE TITLE
Support for CocoaPods 1.13

### ISSUE THREAD

**Issue by SethArchambault** [doublesymmetry/cocoapods-patch#30]
```
$ bundle exec pod install
Calling `DidYouMean::SPELL_CHECKERS.merge!(error_name => spell_checker)' has been deprecated. Please call `DidYouMean.correct_error(error_name, spell_checker)' instead.
You have requested:
  cocoapods ~> 1.13

The bundle currently has cocoapods locked at 1.12.1.
Try running `bundle update cocoapods`

If you are updating multiple gems in your Gemfile at once,
try passing them all to `bundle update`
Run `bundle install` to install missing gems.

$ bundle update cocoapods
Calling `DidYouMean::SPELL_CHECKERS.merge!(error_name => spell_checker)' has been deprecated. Please call `DidYouMean.correct_error(error_name, spell_checker)' instead.
Fetching gem metadata from https://rubygems.org/........
Resolving dependencies...
Bundler could not find compatible versions for gem "cocoapods":
  In Gemfile:
    cocoapods (~> 1.13)

    cocoapods-patch (>= 1.2.0) was resolved to 1.2.0, which depends on
      cocoapods (~> 1.12.0)

```

**Comment by lmcjt37** [comment]
Is there any scope to try and get this PR vetted and merged to support this issue? 

https://github.com/doublesymmetry/cocoapods-patch/pull/29

**Comment by dcvz** [comment]
cocoapods-patch version 1.3.0 has now been published which has 1.13 support
